HelpSource/Classes/Object.schelp

```diff
@@ -691,9 +1050,142 @@ code::
 
 method::dump
 
-Print a detailed low level representation of the receiver to the post window.
+Print a detailed low-level representation of the receiver to the post window.
```

Comment:
Again please try to match the style of other documentation and avoid using nested lists and tables.

Reply:
done!